pkgs/by-name/ne/nekoray/package.nix

```diff
@@ -3,14 +3,17 @@
   stdenv,
   fetchFromGitHub,
   libsForQt5,
```

Comment:
It seems upstream has switched to Qt6. Maybe we should follow this change.

Reply:
Agreed.